app/assets/javascripts/initializers/initializeCommentPreview.js

```diff
@@ -1,50 +1,55 @@
-function initializeCommentPreview() {
-  if (document.getElementById('preview-button')) {
-    document.getElementById('preview-button').onclick = handleCommentPreview;
-  }
-}
-
-function handleCommentPreview(e) {
-  e.preventDefault();
-  var form = e.target.parentElement.parentElement;
-  var textArea = form.querySelector('textarea');
-  if (textArea.value !== '') {
-    var previewDiv = form.querySelector('.comment-preview-div');
-    var previewButton = form.querySelector('.comment-action-preview');
-    if (previewButton.innerHTML.indexOf('PREVIEW') > -1) {
-      textArea.classList.toggle('preview-loading');
-      getAndShowPreview(previewDiv, textArea);
-      previewButton.innerHTML = 'MARKDOWN';
-    } else {
-      previewDiv.classList.toggle('preview-toggle');
-      textArea.classList.toggle('preview-toggle');
-      previewButton.innerHTML = 'PREVIEW';
-    }
+function getAndShowPreview(markdownPreviewPane, markdownEditor) {
```

Comment:
The ordering of the functions changed so that eslint doesn't complain about using a function before it's defined.